.github/workflows/build-container.yml

```diff
@@ -563,6 +563,11 @@ jobs:
           docker load < ceph.tar
           docker load < bdevperf.tar
 
+      - name: Adjust configuration
+        run: |
+          sed -i 's/#max_subsystems = 128/max_subsystems = 256/' ceph-nvmeof.conf
+          sed -i 's/#max_namespaces = 1024/max_namespaces = 2044/' ceph-nvmeof.conf
+
```

Comment:
@gbregman I am not sure we need this modification, unless I miss something  `Adjust configuration` step should be removed.

Reply:
We need to reduce the max_namespaces for this test